src/containers/EventsContainer/EventsContainer.tsx

```diff
@@ -18,11 +20,13 @@ export default class EventsContainer extends Component<
     noNextMeetupEvent: false,
     pastMeetupEvents: [],
     recentTweets: [],
+    videos: [],
   };
 
-  public componentDidMount() {
+  public async componentDidMount() {
```

Comment:
Is async not only needed if there is await?

Reply:
Removed!